### PR TITLE
[antd_v3.x.x] Added definitions for Typography, Text, Title and Paragraph

### PR DESCRIPTION
**Pull request by zomars**
From [the docs](https://ant.design/components/typography/)

- Links to documentation:
- Link to GitHub or NPM: 
- Type of contribution: new definition | addition | fix | refactor

Other notes:

### CODE REVIEW

File: definitions/npm/antd_v3.x.x/flow_v0.25.x-/antd_v3.x.x.js [hunk 1]
```diff
@@ -586,6 +586,37 @@ declare module "antd" {
...
+  declare export class Typography extends React$Component<{}> {
+    static Title: typeof Title;
+    static Text: typeof Text;
+    static Paragraph: typeof Paragraph;
+  }
+
+  // These props are shared by Paragraph, Title, and Text
+  declare type TypographySharedProps = {
+    code: boolean,
+    copyable: boolean | { text: string, onCopy: Function },
```
Comment (pascalduez) [inline definitions/npm/antd_v3.x.x/flow_v0.25.x-/antd_v3.x.x.js]: `Function` is now `any`, could be replaced with `() => void` in this case.

File: definitions/npm/antd_v3.x.x/flow_v0.25.x-/antd_v3.x.x.js [hunk 2]
```diff
@@ -586,6 +586,37 @@ declare module "antd" {
...
+    delete: boolean,
+    disabled: boolean,
+    editable: boolean | { editing: boolean, onStart: Function, onChange: (string) => void },
```
Comment (pascalduez) [inline definitions/npm/antd_v3.x.x/flow_v0.25.x-/antd_v3.x.x.js]: `Function` is now `any`, could be replaced with `() => void` in this case.

**Comment by pascalduez** [comment]
Could you please add a few simple tests for those new components?

**Comment by zomars** [comment]
Done! Is this okay?

File: definitions/npm/antd_v3.x.x/flow_v0.25.x-/antd_v3.x.x.js [hunk 3]
```diff
@@ -586,6 +586,37 @@ declare module "antd" {
...
+    copyable?: boolean | { text: string, onCopy: () => void },
+    delete?: boolean,
+    disabled?: boolean,
+    editable?: boolean | { editing: boolean, onStart: () => void, onChange: (string) => void },
+    ellipsis?: boolean,
+    mark?: boolean,
+    underline?: boolean,
+    onChange?: (string) => void,
+    strong?: boolean,
+    type?: 'secondary' |' warning' | 'danger',
```
Comment (pascalduez) [inline definitions/npm/antd_v3.x.x/flow_v0.25.x-/antd_v3.x.x.js]:
```suggestion
    type?: 'secondary' | 'warning' | 'danger',
```

**Comment by pascalduez** [comment]
Closing due to inactivity, raise a hand if you still want to work on this (fix and rebase).

**Pull request closed without merging**